spec/dummy/client/webpack.client.rails.build.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = merge(config, {
     filename: '[name]-[hash].js',
 
     // Leading slash is necessary
-    publicPath: '/' + webpackPublicOutputDir,
+    publicPath: '/' + webpackPublicOutputDir + '/',
```

Comment:
This was a change @robwise and I made bc not all leading slashes were added in all the right places. (Only in images).  Thus, we removed the leading slashes before images in webpack.common.config.js